src/main/java/ttt/Board.java

```diff
@@ -2,149 +2,108 @@
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 public class Board {
 
-    private String[] grid = new String[9];
-    private String status;
-    private String emptyCell;
-    private String winningMark;
+    private Marks[] board = new Marks[9];
+    private final Marks emptyCell;
 
     public Board() {
-        this.emptyCell = " ";
-        this.winningMark = null;
+        this.emptyCell = Marks.CLEAR;
         for (int cell = 0; cell < 9; cell++) {
-            grid[cell] = emptyCell;
+            board[cell] = emptyCell;
         }
     }
 
-    public void placeMark(String symbol, int location) {
-        if (!isValidLocation(location)) {
-            this.status = "invalid location";
-        } else if (isCellTaken(location)) {
-            this.status = "taken";
-        } else {
-            grid[location] = symbol;
-            status = "mark placed";
-        }
-    }
-
-    public String getStatus() {
-        return status;
+    public void placeMark(Marks mark, int location) {
+        board[location] = mark;
     }
 
     public int size() {
-        return grid.length;
-    }
-
-    public String get(int cell) {
-        return grid[cell];
-    }
-
-    public String getWinningMark() {
-        return winningMark;
+        return board.length;
     }
 
-    public boolean isAWinFor(String mark) {
-        return isWon() && winningMark.equals(mark);
+    public Marks getMarkAt(int cell) {
+        return board[cell];
     }
 
-    public void clear (int location) {
-        grid[location] = emptyCell;
+    public void clear(int location) {
+        board[location] = emptyCell;
     }
 
     public List<Integer> availableMoves() {
         List<Integer> moves = new ArrayList<>();
-        for (int cell = 0; cell < grid.length; cell ++) {
-            if (grid[cell].equals(emptyCell)) {
-               moves.add(cell);
-            }
+        for (int cell = 0; cell < board.length; cell++) {
+            if (board[cell].equals(emptyCell)) moves.add(cell);
```

Comment:
As above. You don't need to use `.equals(...)` for the enums.
You can directly use `==` as underneath, they are just ints (_more or less.._)